js/tootils/src/render.ts

```diff
@@ -69,6 +83,7 @@ export async function render<
 		target,
 		//@ts-ignore
 		props: {
+			loading_status,
```

Comment:
I've updated the test utils so we don't need to pass `loading_status` to every component in tests. We _can_ pass one in if we want to test it for any reason but a default is passed now.

Reply:
Nice!